node-notes-cli/app.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const jsonObject = require('./data.json');
+const command = process.argv[2].toLowerCase();
+const firstArg = process.argv[3];
+const secondArg = process.argv[4];
+const notes = jsonObject.notes;
+const nextId = jsonObject.nextId;
+
+fs.readFile('data.json', 'utf8', (err, json) => {
+  if (err) throw err;
```

Comment:
You can completely skip this readFile step if you are using `require()` to load the data.